dev-python/regex/regex-2019.12.20.recipe

```diff
@@ -0,0 +1,64 @@
+SUMMARY="All things regular expression: implemented in python"
+DESCRIPTION="Regular expression module, to replace re"
+HOMEPAGE="https://pypi.org/project/regex/"
+COPYRIGHT="2013-2019 Matthew Barnett"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://pypi.org/packages/source/r/regex/regex-$portVersion.tar.gz"
+CHECKSUM_SHA256="106e25a841921d8259dcef2a42786caae35bc750fb996f830065b3dfaa67b77e"
+SOURCE_DIR="regex-2019.12.20"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	$portName = $portVersion
```

Comment:
My bad, you  are missing some $secondaryArchSuffix in the recipe (4 places)

Reply:
like now?

Reply:
$portName should be $secondaryArchSuffix too